Automattic/woocommerce-payments
# Updated the rest api docs for `deposits/overview-all`

Fixes #8346

#### Changes proposed in this Pull Request
- Added `default_external_accounts` element to REST API docs for `deposits/overview-all`

#### Testing instructions
- Verify the markdown
- Can build docs locally to verify the REST API docs

-------------------


**Post merge**
